forge-gui/res/cardsfolder/upcoming/soundwave_sonic_spy_soundwave_superior_captain.txt

```diff
@@ -0,0 +1,31 @@
+Name:Soundwave, Sonic Spy
+ManaCost:1 W U B
+Types:Legendary Artifact Creature Robot
+PT:5/4
+K:More Than Meets the Eye:1 W U B
+T:Mode$ DamageDoneOnce | CombatDamage$ True | ValidSource$ Creature.token+YouCtrl | TriggerZones$ Battlefield | ValidTarget$ Player | Execute$ TrigCast | TriggerDescription$ nWhenever one or more creature tokens you control deal combat damage to a player, exile target instant or sorcery card with mana value equal to the damage dealt from their graveyard. Copy it. You may cast the copy without paying its mana cost. If you do, convert NICKNAME.
+SVar:TrigCast:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | TgtPrompt$ Choose target instant or sorcery card in that player' graveyard | ValidTgts$ Instant.cmcLEX+OwnedBy TriggeredTarget,Sorcery.cmcLEX+OwnedBy TriggeredTarget | RememberChanged$ True | SubAbility$ DBPlay
+SVar:DBPlay:DB$ Play | Valid$ Card.IsRemembered | ValidZone$ Exile | Controller$ You | CopyCard$ True | WithoutManaCost$ True | ValidSA$ Spell | Optional$ True | RememberCast True | SubAbility$ DBConvert
```

Comment:
RememberCast doesn't exist
you may want to use `ImprintPlayed` so it doesn't collide